beacon-chain/blockchain/options.go

```diff
@@ -208,20 +211,42 @@ func WithBlobStorage(b *filesystem.BlobStorage) Option {
 	}
 }
 
+// WithDataColumnStorage sets the data column storage backend for the blockchain service.
+func WithDataColumnStorage(b *filesystem.DataColumnStorage) Option {
+	return func(s *Service) error {
+		s.dataColumnStorage = b
+		return nil
+	}
+}
+
 func WithSyncChecker(checker Checker) Option {
 	return func(s *Service) error {
 		s.cfg.SyncChecker = checker
 		return nil
 	}
 }
 
+func WithCustodyInfo(custodyInfo *peerdas.CustodyInfo) Option {
```

Comment:
Want to add godoc here?

Reply:
Fixed in c909a3d5d87e5868f7e9c5ffc45c9957c630b239.